syntax = "proto3";

package authService;


service auth {
  rpc GetToken(user_password_request) returns (token_answer) {}
  rpc CheckToken(check_token_request) returns (ok_answer) {}
  rpc RegisterUser(user_password_request) returns (ok_answer) {}
}


message user_password_request {
  string user = 1;
  string password = 2;
}

message check_token_request {
  string token = 1;
}

message ok_answer {
  int32 ok_code = 1;
}

message token_answer {
  string token = 1;
}